CHANGELOG.md

```diff
@@ -3,11 +3,16 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 # [v1.3.0](unreleased)
 ### Notes
-- This release supports for API 800,1000,1200  for resource listed below.
+- This release supports to API 800,1000,1200 for Hypervisor Manager and Server Certificate resources.
```

Comment:
Mention the 1600 version as well.

Reply:
Updated!

Reply:
@AnushaY1916 
Reverted to same i.e.
"This release supports to API 800,1000,1200 for Hypervisor Manager and Server Certificate resources."
since HM and SC resources for API1600 would be added by venkatesh as planned, so he will add for 1600. Ill leave it till whats done till now.